### DISCUSSION TITLE
Editing the UI

### DISCUSSION THREAD

**Discussion by HoJiYuanEric** [spring-petclinic/spring-petclinic-microservices#209] [Q&A] [answered]
Hi, may I ask for advice on how to edit the UI of this application? 
When I try changing the HTML file in the project, the UI does not seem to reflect it.

Thank you!

**Comment by arey** [comment, marked as answer]
Hi @HoJiYuanEric 
Do you restart the `spring-petclinic-api-gateway` after changing the html files?
